go/mysql/conn.go

```diff
@@ -183,13 +183,36 @@ type Conn struct {
 	// - readEphemeralPacket / recycleReadPacket methods for reads.
 	currentEphemeralPolicy int
 	// TODO (danieltahara): Ultimately get rid of this delineation.
-	currentEphemeralPacket []byte
+	currentEphemeralPacket *[]byte
```

Comment:
Can you add a comment here delineating between this and currentEphemeralBuffer? Might even be worthwhile to rename to currentEphemeralWriteBuffer, currentEphemeralReadBuffer.